pkg/ui/src/views/reports/containers/debug/index.tsx

```diff
@@ -200,6 +200,7 @@ export default function Debug() {
             url="/_admin/v1/rangelog/1?limit=100"
             note="/_admin/v1/rangelog/[range_id]?limit=100"
           />
+          <DebugTableLink name="Settings" url="/_admin/v1/settings" />
```

Comment:
It's probably obvious from context, but it might be a good idea to say "Cluster Settings" here, since we almost always refer to it by that full name.

Reply:
Done. I had debated about that myself but gone the other direction due to the redundancy in context.